### DISCUSSION TITLE
CMU 10-708: Probabilistic Graphical Models - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#640] [Announcements]
# CMU 10-708: Probabilistic Graphical Models - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/CMU10-708/

**Comment by guotree** [comment]
非常赞的资料整理，或许可以加上站在初学者的角度写一写各门课的学习顺序~

**Comment by Cichlider** [comment]
"第三，也是我觉得最困难的一点，就是自律......而这一切，没有学分，没有绩点，没有老师，没有同学，只有一个信念 —— 你在变强。"很难不为之动容。哥们的文笔真好

**Comment by Vitoom** [comment]
最新课程网站: https://andrejristeski.github.io/10708S25/

**Reply by Vitoom** [reply]
20年recording: https://www.cs.cmu.edu/%7Eepxing/Class/10708-20/lectures.html
